.pre-commit-config.yaml

```diff
@@ -11,7 +11,7 @@ repos:
     rev: v1.15.0
     hooks:
       - id: mypy
-        additional_dependencies: [click == 8.1.8, pydantic == 2.10.6]
+        additional_dependencies: [click == 8.1.8, pydantic == 2.10.6, faker>=36.1.1]
```

Comment:
Nice that fixes a known difference between mypy and pre-commit mypy! So this probably closes #236